### ISSUE TITLE
require vs define

### ISSUE THREAD

**Issue by zowers** [pesla/element-queries#1]
why are you using require() instead of define() here?
https://github.com/pesla/element-queries/blob/a18fa033b2aa9cf80670298de8aafa54f33726b9/src/element-queries.js#L10

afaik define() must be used in modules, require() is only for loading entry point from the page

**Comment by pesla** [comment]
@zowers Good point, this acted as a loading entry point, but I forgot to rewrite it. I'll do that later this week. Thanks for notifying!

**Comment by zowers** [comment]
thank you for great library!
